src/fixturelibrary.ts

```diff
@@ -4,13 +4,20 @@ import addFormats from 'ajv-formats';
 import { pathExistsSync, readJSONSync } from 'fs-extra';
 import { FixtureIndex } from './fixtureindex';
 import {
-  fetchOflFixtureDirectory, githubRawFixtureRequest, request,
+  fetchOflFixtureDirectory, githubRawFixtureRequest, githubRawManufacturersRequest, request,
 } from './webhandler';
 import { Fixture } from './types';
 
 import * as schema from './ofl-schema/ofl-fixture.json';
 import fileHandler from './filehandler';
 
+interface Manufacturers { [key:string]: { name?: string } }
+interface MakeModel {
+  key: string;
+  make?: string;
+  model?: string;
```

Comment:
```suggestion
  make: string;
  model: string;
```

They should all have one, no?